tools/pythonpkg/src/map.cpp

```diff
@@ -40,7 +40,7 @@ static py::handle FunctionCall(NumpyResultConversion &conversion, const vector<s
 		throw InvalidInputException("Python error. See above for a stack trace.");
 	}
 
-	py::handle df(df_obj);
+	auto df = py::reinterpret_steal<py::object>(df_obj);
```

Comment:
This also used to be a leak, py::handle does not decrease ref when it goes out of scope